<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogFontSize</class>
 <widget class="QDialog" name="DialogFontSize">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>291</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QListWidget" name="fontList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>100</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Крошечный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Очень маленький</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Маленький</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Мелкий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Стандартный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Крупный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Большой</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Очень большой</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Огромный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Невероятно огромный</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QComboBox" name="cbType">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Размер описания задач</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Размер математических выражений</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="3">
    <widget class="QWebView" name="webPreview">
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogFontSize</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogFontSize</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
